gcp-observability/src/main/proto/grpc/observabilitylog/v1/observabilitylog.proto

```diff
@@ -28,151 +28,99 @@ option java_outer_classname = "ObservabilityLogProto";
 message GrpcLogRecord {
   // List of event types
   enum EventType {
-    GRPC_CALL_UNKNOWN = 0;
```

Comment:
This comment is for the whole file: the change we are making is cross language (proto) but it seems we maintain multiple copies of this file in various language repos. Have we considered sourcing this file from a central place (like grpc-proto)?

Reply:
I did talk about this with @ejona86. And decided to have it in language repos for time being.

Reply:
There's split perspectives about this file. For starters, it is 100% internal; it is just a convenience to produce the JSON. Doug hated using the proto stuff so Go swapped away from using the proto-to-JSON conversion. He doesn't want to see this as a schema or anything like that. C I think was interested in using it still, and Mark preferred the idea of using it as a schema in the design. We'll need to work through that, but just choosing the name for the file is more that we'd do right now.